jekyll/_cci2/rerun-failed-tests-only.adoc

```diff
@@ -224,6 +224,7 @@ You can also add it to your `jest.config.js` file by following these link:https:
 ```
 
 . Update the `glob` command to match your use case. **If your current job is using xref:test-splitting-tutorial#[CircleCI's intelligent test splitting], you must change the `circleci tests split` command to `circleci tests run` with the `--split-by=timings` parameter.**. If you are not using test splitting, `--split-by=timings` can be omitted. Note: you may also use link:https://playwright.dev/docs/test-reporters#junit-reporter[Playwright's built-in flag] (`PLAYWRIGHT_JUNIT_OUTPUT_NAME`) to specify the JUnit XML output directory.  
+. Ensure that you are using the latest version of Playwright.  Version 1.27.1 of Playwright does not output JUnit XML in a format that is compatible with this feature.
```

Comment:
I was wondering if we should specify a minimum working version. We can say something along the lines of "ensure that Playwright is at version v1.34.2 (what worked for that customer) or higher"

Reply:
fair.  i wasnt sure if like 1.28 worked for example.  so i didnt want someone who had 1.28 to feel like they had to upgrade if they didnt actually have to.   but i like your idea more, let's do that